### PR TITLE
(chore) O3-3069 tweak syling for queue table

### PR DESCRIPTION
**Pull request by chibongho**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

This PR applies styling fixes to the service queues table. [Similar fixes](https://github.com/openmrs/openmrs-esm-patient-management/pull/1092) were done to the appointments table.

## Screenshots

Before:
image

After:
image
- Smaller table header
- table header has an expander button to expand all rows

### CODE REVIEW

**Review by denniskigen — APPROVED** [review]
Thanks, @chibongho. Is this table the same table shown on the service queues dashboard in dev3?

CleanShot 2024-04-13 at 12  01 29@2x

Because it appears that this is a different table that is affected by different styles.

File: packages/esm-service-queues-app/src/queue-table/queue-table.component.tsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ function QueueTable({
           <Table {...getTableProps()} className={styles.queueTable} useZebraStyles>
             <TableHead>
               <TableRow>
-                {ExpandedRow && <TableExpandHeader />}
+                {ExpandedRow && <TableExpandHeader enableToggle {...getExpandHeaderProps()} />}
                 {headers.map((header) => (
                   <TableHeader {...getHeaderProps({ header })}>{header.header}</TableHeader>
                 ))}
```
[the remark this answers is not in the document]
Comment (chibongho) [reply packages/esm-service-queues-app/src/queue-table/queue-table.component.tsx:80]: It's behind a feature flag right now, but hopefully won't be soon. I have a [pending PR](https://github.com/openmrs/openmrs-esm-patient-management/pull/1098#issuecomment-2052430424) to make this table be the default one you see when loading the service queue app.

[the remark this answers is not in the document]
Comment (denniskigen) [reply packages/esm-service-queues-app/src/queue-table/queue-table.component.tsx]: Thanks for the context. I've toggled the features flag and seen the queue table in action. 

CleanShot 2024-04-13 at 12  19 38@2x

A couple of other thoughts:

- It's probably worth adding an empty state for the `Coming from` column that gets shown when there's no data. I think we usually default to `--`.
- The table should have a border of `1px solid $ui-03`
- You could

[the remark this answers is not in the document]
Comment (chibongho) [reply packages/esm-service-queues-app/src/queue-table/queue-table.component.tsx]: Thanks @denniskigen. A lot of those are details I don't see until they are pointed out. I will merge this in and address those in a separate PR.

**Comment by denniskigen** [comment]
Just occurred to me that this is a (feat) type change - not a (chore).

**Pull request merged**